promote-release-to-box.py

```diff
@@ -344,15 +346,18 @@ def upload_to_box(client, artifacts_to_release, artifact_to_box_path):
 
 
     ###### Upload to box ##########
+    # determine the build suffix 9.5.1.0-124
+    build_suffix = build_version + '-' + build_number
+
     # downloads artifacts
     builds_output_json, artifacts_in_build_info = get_artifact_info_json(build_name, build_number, rt_auth=rt_auth, rt_base_url=rt_base_url)
-    file_folder_dict = process_manifest_yaml(get_manifest_yaml(build_number, manifest_file = manifest_file_path))
+    file_folder_dict = process_manifest_yaml(get_manifest_yaml(build_suffix, manifest_file = manifest_file_path))
     artifacts_to_release = get_manifest_buildinfo_intersect(file_folder_dict, builds_output_json)
     downloaded_artifacts = download_artifacts_v3(artifacts_to_release, builds_output_json, auth=rt_auth, rt_base_url = rt_base_url)
 
     # if there are no files to deploy, exit process
     if not artifacts_to_release:
-        logging.WARNING(f'There are no artifacts to be promoted with build name:{build_name}, build number: {build_number}')
+        logging.warning(f'There are no artifacts to be promoted with build name:{build_name}, build number: {build_number}, build version {build_version}')
```

Comment:
What's the correct case for the logging level here?
Before you had all caps WARNING, but now I see LC warning. Perhaps it is case insensitive?
Also, shouldn't this be logging.ERROR?

Reply:
It should be lower case for logging, I made a mistake.
Sure, I'll change warning level to error since it's not matching anything.